Z9264f new platform changes

paavaanan opened the pull request:
**- Added sai.profile, minigraph and port_config.ini for 64x100 z9264f platform.**
**- Added psuutil support script with ipmi support.**
**- Added  scripts for psuutil, sfputil, eeprom.**
**- FPGA driver support is added in sonic-platform-modules-dell repository. Associated Makefiles and init 
scripts are updated accordingly.**

lguohan committed:
[baseimage]: use debian 4.9.0-3 kernel

lguohan committed:
[baseimage]: upgrade initramfs to 0.130

lguohan committed:
[baseimage]: install systemd-sysv in the base image

lguohan committed:
[baseimage]: build root filesystem via overlay fs instead of aufs

lguohan committed:
[baseimage]: Disable consistent network device naming

consistent network device naming is enabled in debian 9 kernel by default

lguohan committed:
[baseimage]: update igb driver to 5.3.5.10

lguohan committed:
[temp]: disable building platform drivers tempoerarily

lguohan committed:
[libnl]: upgrade libnl3 from 3.2.27-1 to 3.2.27-2

lguohan committed:
[docker-syncd]: install kmod in docker-syncd-brcm without prompt

lguohan committed:
[opennsl]: use opennsl kernel module based on kernel 4.9.0-3

lguohan committed:
[baseimage]: let docker in base image use overlay fs instead of aufs

lguohan committed:
[baseimage]: update base image from jessie to stretch

lguohan committed:
[sonic-slave]: add deps for build initramfs 0.130

lguohan committed:
[build]: insert overlay kmod for base image build

lguohan committed:
[build]: build stretch-based sonic slave docker

lguohan committed:
[build]: sonic-slave-stretch- sha both Dockerfile and Dockerfile.user

lguohan committed:
[kernel]: fix linux-headers common deb file name

lguohan committed:
update build instructions

lguohan committed:
[kernel]: update kernel submodule and remove standalone igb driver

lguohan committed:
[baseimage]: install acl package

lguohan committed:
[baseimage]: install tacacs dependencies

marian-pritsak committed:
[Mellanox]: Upgrade MFT package to 4.8.26 (#3)

This version is compatible with Linux kernel 4.9

Signed-off-by: marian-pritsak <marianp@mellanox.com>

marian-pritsak committed:
[Mellanox]: Update recipe for hw-management (#4)

Update package revision
Create submodule for sources

Signed-off-by: marian-pritsak <marianp@mellanox.com>

lguohan committed:
[platform]: enable platform driver for dell s6000 (#5)

lguohan committed:
[kernel]: upgrade linux kernel to 4.9.0-5 (4.9.65-3+deb9u2) (#8)

lguohan committed:
[ixgbe]: compile and install ixgbe to 4.9.0-5 kernel

marian-pritsak committed:
[mellanox]: Use KVERSION variable in mft recipe (#11)

Signed-off-by: marian-pritsak <marianp@mellanox.com>

marian-pritsak committed:
[mellanox]: Adapt to new hw-management package (#12)

Use single start script for all platforms and remove symbolic links
Change path to system eeprom

Signed-off-by: marian-pritsak <marianp@mellanox.com>

lguohan committed:
[devices]: enable platform modules for various platforms

lguohan committed:
[dell]: update to stretch-kernel based dell modules

lguohan committed:
[platform]: incorporate sonic-platform-modules-cel into sonic buildimage repo

lguohan committed:
[devices]: enable sonic-platform-modules-cel

pwisutti committed:
[celestica] change to use lm75 standard driver (#19)

nealtai committed:
[Delta]: Modify delta-ag9032v1 driver for kernel 4.9 (#20)

Signed-off-by: neal tai <neal.tai@deltaww.com>

lguohan committed:
[build]: add stretch target to build in stretch docker

lguohan committed:
[build]: build initramfs, libwrap, tacacs packages under stretch docker

initramfs/libwrap/tacacs packages are installed on stretch base image

lguohan committed:
[broadcom]: update opennsl module to 3.4.1.11-1

lguohan committed:
[bug fix]: fix delta ag9032v1 driver compilation. disable ag9064

lguohan committed:
[baseimage]: move update initramfs to later stage

some platform drivers install blacklist.conf in /etc/modprobe.d.
Those configuration should be proprogated into initramfs to avoid
loading those blacklisted driver.

Staphylo committed:
[device]: Enable arista drivers for sonic-linux-kernel 4.9 (#21)

* Enable arista drivers

* Add vfat ascii charset in initramfs

* Update boot0.j2 for 4.9 kernel

* Fix i2c offsets in sensors.conf

* Bump sonic-platform-modules-arista submodule

Commit:
[device]: add the new platform ag9064 (#23)


Signed-off-by: hans <hans.tseng@deltaww.com>

Commit:
[devices]: Added new Mellanox MSN2010 platform (#24)

Commit:
Added new platform support DellEMC - Z92264f - 64x100

lguohan committed:
[devices]: disable compiling ag9064 and ag5648

Signed-off-by: Guohan Lu <gulv@microsoft.com>

lguohan committed:
[devices]: disable compiling ag9064 and ag5648

Signed-off-by: Guohan Lu <gulv@microsoft.com>

StanleyCi committed:
[devices]: Modified and fixed delta ag5648 modules. (#30)

* Modified and fixed delta ag5648 modules
1. modified and fixed delta ag5648 modules.
2. enable build ag5648.

Signed-off-by: neal tai <neal.tai@deltaww.com>

* Enable build ag9064

Signed-off-by: neal tai <neal.tai@deltaww.com>

Commit:
[devices]: modified and fixed delta ag9064 modules (#28)

2. enable build ag9064

Signed-off-by: neal tai <neal.tai@deltaww.com>

mykolaf committed:
[mellanox]: fix SAI build makefile (#29)

yurypm committed:
[baseimage]: Add derived package to initramfs-tools (#33)

Commit:
Includes changes with Makefiles, sfputil, eeprom and default minigraph

Staphylo committed:
Add support 4.9 support for 7260CX3 (#34)

lguohan commented on device/dell/x86_64-dellemc_z9264f_c3538-r0/plugins/psuutil.py:
```diff
@@ -0,0 +1,97 @@
+
+
+Z9264F_MAX_PSUS = 2
+IPMI_SENSOR_DATA = "ipmitool sdr list"
```
how does ipmitool communicate with the platform controller?

lguohan replied:
2. the psuutil is also invoked in snmp docker to provide psu status snmp query, can this psuutil.py be run in the snmp docker?

StanleyCi committed:
[delta]: Enable ag5648,  et-6248brb, ag9064 and fix the build error of ag5648 PSU module. (#32)

* 1. Fixed delta ag5648 PSU modules.
2. Enable build ag5648.

Signed-off-by: neal tai <neal.tai@deltaww.com>

* 1. Modified the kernel depends for ag5648.
2. Modified platform-modules-ag5648.service for install ag5648.

Signed-off-by: neal tai <neal.tai@deltaww.com>

* Enable the ag9064.

Signed-off-by: Stanley Chi <stanley.chi@deltaww.com>

* Add Delta platform et-6248brb

Signed-off-by: neal tai <neal.tai@deltaww.com>

Commit:
Led support for Z9264f platform

Commit:
Includes changes on default minigraph

Commit:
ipmitool implementation in pmon docker. platform_sensors script is inclued in pmon startup

lguohan committed:
Merge branch 'stretch' into Z9264f_new_platform_changes

lguohan commented:
can you do a rebase and leave only the necessarily change

lguohan commented:
Updated in another PR.
